app/src/app/api/v0/city/[city]/inventory/[year]/sector/[sector]/subsector/[subsector]/subcategory/route.ts

```diff
@@ -0,0 +1,22 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { createSubCategory, createSubSectorRequest } from "@/util/validation";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+import { randomUUID } from "node:crypto";
+
+export const POST = apiHandler(async (req: NextRequest, { params }) => {
+  const body = createSubCategory.parse(await req.json());
```

Comment:
I think tbh it's better to have the validation definitions in the route file where it's used (if it's only used in one route).

So that's a refactoring we could do, but not 100% required right now.